js/app/test/video_component_events.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { test, expect } from "@gradio/tootils";
```

Comment:
neeeded to add the .ogg file because chromium can't play the mp4 file with h264 codec cause its proprietary. Took me forever to figure out why the play event wasn't triggering 😅